server/server.js

```diff
@@ -0,0 +1,30 @@
+// Server
+const express = require('express')
+const app = express()
+const port = 5000
+const path = require('path');
+
+
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`)
+})
+
+// Pages
```

Comment:
could have added an array for the keywords so you only need 1 app.get('/:id',(req, res) for each keyword